libraries/botframework-config/src/botConfiguration.ts

```diff
@@ -263,48 +314,50 @@ export class BotConfiguration implements Partial<IBotConfiguration> {
                 throw err;
             }
 
-            return service;
+            // return service;
```

Comment:
Why is this commented out?

Reply:
It didn't look like it was doing anything there, but I wasn't 100% sure.

It is returning inside a catch block, inside a function with a void return type.

Reply:
@tomlm, can you comment on this? The `return service` doesn't look like it's supposed to be used, based on successful completion of the first `try` block and the lack of a return value there.

Reply:
This looks like it should be deleted. If we can't get confirmation from @tomlm, consider leaving it as-is, so we can get the rest of this PR-Of-Happy-Linting commited!!

Reply:
@benbrown, I concur with Chris, let's just revert this line for now to leave the current behavior and get this in.